libsql-server/src/hrana/ws/session.rs

```diff
@@ -102,12 +114,23 @@ pub(super) async fn handle_repeated_hello<F: MakeNamespace>(
     }
     let namespace_jwt_key = server
         .namespaces
-        .with(namespace, |ns| ns.jwt_key())
+        .with(namespace.clone(), |ns| ns.jwt_key())
         .await??;
+
+    // Convert jwt token into a HeaderValue to be compatible with UserAuthStrategy
+    let user_credential = jwt
```

Comment:
unfortunate but I think this is fine for now, WS is not our primary protocol anyways for hrana so this I think is acceptable for now.

I think the only thing this really poses is should we accept a String rather than a HeaderValue in the auth trait?

Reply:
I like the idea of `user_credential` being an `Option<String>` though a couple of things off the top of my head I think we'd need to account for:

* In HTTP context, do we pass the entire `Authorization` header value?
* In this WS context, do we still append an Authorization scheme or does the strategy need to do some extra homework to determine if it's being given a header value or something else?

Reply:
I think it would make sense to strip the prefix at the http/ws level and then only pass through the "token" to the Auth layer? I feel like that would make the most sense. What do you think? That way we can just set this as a string/bytes and not leak HeaderValue through to the auth layer.